recipes/billingegroup-meta/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "billingegroup-meta" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True  # [py36 or win]
+  number: 0
+
+requirements:
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - seaborn
+    - fabio
+    - pims
+    - yaml
+    - pandas
+    - jupyter
+    - pyFAI  # [linux]
+
+test:
+  imports:
+    - matplotlib
+    - pyFAI
+
+about:
+  home: https://thebillingegroup.com/
+  license: MIT
```

Comment:
Since this is a metapackage I really don't have a good sense of what the license should be. 

IANAL.

It's not even clear *what* you're licensing. Are you licensing the list of packages that make up this metapackage? Are you trying to indicate what all of the underlying licenses are for *all* the projects that this pulls in? Are you licensing this metapackage specification?